### PR TITLE
Additional Testing

### PR DESCRIPTION
**Pull request by timshell**

### CODE REVIEW

**Comment by timshell** [comment]
also should be good to go @pavanramkumar

**Comment by timshell** [comment]
should be good to go now @pavanramkumar

File: tests/test_neurovis.py [hunk 1]
```diff
@@ -1,21 +1,22 @@
-import matplotlib.pyplot as p
-p.switch_backend('Agg')
-from spykes.neurovis import NeuroVis
-from nose.tools import assert_true, assert_equal, assert_raises
+from nose.tools import assert_true, assert_equal
 import numpy as np
 import pandas as pd
-
+import matplotlib.pyplot as p
```
Comment (jasmainak) [inline tests/test_neurovis.py]: normally you call it `plt` not `p`

File: tests/test_neurovis.py [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
+p.switch_backend('Agg')
```
Comment (jasmainak) [inline tests/test_neurovis.py]: Qt4Agg

File: tests/test_popvis.py [hunk 3]
```diff
@@ -1,25 +1,27 @@
-import matplotlib.pyplot as p
-p.switch_backend('Agg')
-from spykes.popvis import PopVis
-from spykes.neurovis import NeuroVis
 from nose.tools import assert_true, assert_equal, assert_raises
 import numpy as np
 import pandas as pd
+import matplotlib.pyplot as p
+p.switch_backend('Agg')
```
Comment (jasmainak) [inline tests/test_popvis.py]: same comments here

File: tests/test_strf.py [hunk 4]
```diff
@@ -1,8 +1,9 @@
-import matplotlib.pyplot as p
-p.switch_backend('Agg')
-from spykes.strf import STRF
 from nose.tools import assert_equal
 import numpy as np
+import matplotlib.pyplot as p
+p.switch_backend('Agg')
```
Comment (jasmainak) [inline tests/test_strf.py]: actually this import should be at the top of the file for it to work ?

Comment (timshell) [inline tests/test_strf.py]: I believe it just needs to be before the import for spykes. It is currently working, though, as  the Travis is building

**Comment by timshell** [comment]
Should be good to go @jasmainak . Backend must be Agg, not Qt4Agg for Travis to work. Other changes have been made

**Comment by jasmainak** [comment]
okay cool, sounds good!

**Comment by timshell** [comment]
cc @pavanramkumar should actually be good to go now

**Comment by timshell** [comment]
cc @pavanramkumar

**Comment by codekansas** [comment]
Hi, I figured I'd re-open this. I'm taking over maintenance from Pavan. Looking through the PR, one bit is that I removed the random scikit learn dependency from one of the tests - do you still want to merge this PR? If so, could you update it to be compatible? I'll take a look afterwards!

**Comment by timshell** [comment]
Hey @codekansas sorry for the late reply. The work you're doing now looks awesome! I'm actually a bit swamped with work for the next week or two, but I can definitely fix this afterwards. Super excited to see where you take Spykes!

**Comment by timshell** [comment]
Should be good to go @codekansas

File: tests/plot/test_neurovis.py [hunk 5]
```diff
@@ -16,13 +15,15 @@
 
 def test_neurovis():
```
Comment (codekansas) [inline tests/plot/test_neurovis.py]: probably should remove this extra newline

File: tests/plot/test_popvis.py [hunk 6]
```diff
@@ -17,6 +17,8 @@
 
 def test_popvis():
```
Comment (codekansas) [inline tests/plot/test_popvis.py]: ditto

**Pull request merged**